### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by signo88**
Added the italian translation for some labels

### CODE REVIEW

**Comment by dEBRUYNE-1** [comment]
Thanks for the PR! Let's leave this open until other native speakers have verified the translation.

**Comment by signo88** [comment]
Ok..

In the next days i will improve the translation.



2016-11-08 15:58 GMT+01:00 dEBRUYNE-1 <notifications@github.com>:

File: translations/monero-core_it.ts [hunk 1]
```diff
@@ -22,17 +22,17 @@
...
         <location filename="../pages/AddressBook.qml" line="77"/>
         <source>Payment ID &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>ID Pagamento &lt;font size=&apos;2&apos;&gt;(Opzionale)&lt;/font&gt;</translation>
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="79"/>
         <source>&lt;b&gt;Payment ID&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;A unique user name used in&lt;br/&gt;the address book. It is not a&lt;br/&gt;transfer of information sent&lt;br/&gt;during thevtransfer</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;ID PAgameto&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;Utente unico usato nella&lt;br/&gt;rubrica.</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: ID Pagamento

File: translations/monero-core_it.ts [hunk 2]
```diff
@@ -22,17 +22,17 @@
...
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="99"/>
         <source>Description &lt;font size=&apos;2&apos;&gt;(Local database)&lt;/font&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>Desscrizione &lt;font size=&apos;2&apos;&gt;(Database locale)&lt;/font&gt;</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Descrizione

File: translations/monero-core_it.ts [hunk 3]
```diff
@@ -159,89 +159,89 @@
...
         <location filename="../pages/History.qml" line="65"/>
         <source>Filter transaction history</source>
-        <translation type="unfinished"></translation>
+        <translation>Filtra lo storico transazioni</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="102"/>
         <source>&lt;b&gt;Payment ID&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;A unique user name used in&lt;br/&gt;the address book. It is not a&lt;br/&gt;transfer of information sent&lt;br/&gt;during thevtransfer</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;ID PAgameto&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;Utente unico usato nella&lt;br/&gt;rubrica.</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: ID Pagamento

File: translations/monero-core_it.ts [hunk 4]
```diff
@@ -159,89 +159,89 @@
...
         <location filename="../pages/History.qml" line="296"/>
         <source>To</source>
-        <translation type="unfinished"></translation>
+        <translation>A</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="192"/>
         <source>FILTER</source>
-        <translation type="unfinished"></translation>
+        <translation>FLITRA</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: FILTRA

File: translations/monero-core_it.ts [hunk 5]
```diff
@@ -251,7 +251,7 @@
     <message>
         <location filename="../LeftPanel.qml" line="142"/>
         <source>Unlocked balance</source>
-        <translation type="unfinished"></translation>
+        <translation>Totale bloccato</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: bloccato o sbloccato?

File: translations/monero-core_it.ts [hunk 6]
```diff
@@ -261,137 +261,137 @@
...
         <location filename="../MiddlePanel.qml" line="180"/>
         <source>Balance:</source>
-        <translation type="unfinished"></translation>
+        <translation>Totale</translation>
     </message>
     <message>
         <location filename="../MiddlePanel.qml" line="218"/>
         <source>Unlocked Balance:</source>
-        <translation type="unfinished"></translation>
+        <translation>Totale Bloccato:</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Bloccato o Sbloccato?

File: translations/monero-core_it.ts [hunk 7]
```diff
@@ -261,137 +261,137 @@
...
         <location filename="../components/NetworkStatusItem.qml" line="58"/>
         <source>Network status</source>
-        <translation type="unfinished"></translation>
+        <translation>Stato Rete</translation>
     </message>
     <message>
         <location filename="../components/NetworkStatusItem.qml" line="66"/>
         <source>Connected</source>
-        <translation type="unfinished"></translation>
+        <translation>Conesso</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Connesso

File: translations/monero-core_it.ts [hunk 8]
```diff
@@ -261,137 +261,137 @@
...
     </message>
     <message>
         <location filename="../components/NetworkStatusItem.qml" line="66"/>
         <source>Disconnected</source>
-        <translation type="unfinished"></translation>
+        <translation>Disconesso</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Disconnesso

File: translations/monero-core_it.ts [hunk 9]
```diff
@@ -261,137 +261,137 @@
...
     </message>
 </context>
 <context>
     <name>PasswordDialog</name>
     <message>
         <location filename="../components/PasswordDialog.qml" line="46"/>
         <source>Please enter wallet password</source>
-        <translation type="unfinished"></translation>
+        <translation>Inserisci la passowrd del tuo portafoglio</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: password

File: translations/monero-core_it.ts [hunk 10]
```diff
@@ -261,137 +261,137 @@
...
         <location filename="../pages/Receive.qml" line="149"/>
         <source>Payment ID here</source>
-        <translation type="unfinished"></translation>
+        <translation>ID Pagamento</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="173"/>
         <source>Generate</source>
-        <translation type="unfinished"></translation>
+        <translation>Generato</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Genera?

File: translations/monero-core_it.ts [hunk 11]
```diff
@@ -466,73 +466,73 @@
...
         <location filename="../components/TickDelegate.qml" line="55"/>
         <source>LOW</source>
-        <translation type="unfinished"></translation>
+        <translation>BASSA</translation>
     </message>
     <message>
         <location filename="../components/TickDelegate.qml" line="56"/>
         <source>MEDIUM</source>
-        <translation type="unfinished"></translation>
+        <translation>MEDIO</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: MEDIA

File: translations/monero-core_it.ts [hunk 12]
```diff
@@ -542,301 +542,302 @@
...
         <location filename="../wizard/WizardConfigure.qml" line="97"/>
         <source>Kickstart the Monero blockchain?</source>
-        <translation type="unfinished"></translation>
+        <translation>Vuoi avviare la blockchain di Monero?</translation>
     </message>
     <message>
         <location filename="../wizard/WizardConfigure.qml" line="115"/>
         <source>It is very important to write it down as this is the only backup you will need for your wallet. You will be asked to confirm the seed in the next screen to ensure it has copied down correctly.</source>
-        <translation type="unfinished"></translation>
+        <translation>à veramente importante che ti salvi queste parole in quanto sono l&apos;unico backup che serve per il tuo portafoglio. Ti sarà chiesto di confermarle nella prossima schermata per essere sicuri che tu le abbia copiate correttamente.</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: è

File: translations/monero-core_it.ts [hunk 13]
```diff
@@ -542,301 +542,302 @@
...
         <location filename="../wizard/WizardConfigure.qml" line="127"/>
         <source>Enable disk conservation mode?</source>
-        <translation type="unfinished"></translation>
+        <translation>Abilita la modalità di conservazione del disco?</translation>
     </message>
     <message>
         <location filename="../wizard/WizardConfigure.qml" line="145"/>
         <source>Disk conservation mode uses substantially less disk-space, but the same amount of bandwidth as a regular Monero instance. However, storing the full blockchain is beneficial to the security of the Monero network. If you are on a device with limited disk space, then this option is appropriate for you.</source>
-        <translation type="unfinished"></translation>
+        <translation>La modalità conservazione del disco usa meno spazio, ma la stessa quantità di banda coem una normale instanza Monero. In ogni caso, conservare la blockchain completa è utile alla sicurezza della rete di Monero. Se sei in un device con spazio sul disco limitato allora questa opzione fa al caso tuo.</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: come

File: translations/monero-core_it.ts [hunk 14]
```diff
@@ -542,301 +542,302 @@
...
         <location filename="../wizard/WizardOptions.qml" line="89"/>
         <source>Please select one of the following options:</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleziona una delle seguenti opzioni:</translation>
     </message>
     <message>
         <location filename="../wizard/WizardOptions.qml" line="129"/>
         <source>This is my first time, I want to&lt;br/&gt;create a new account</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa la mia prima volta, vorrei&lt;br/&gt;creare un nuovo account</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Questa è...

File: translations/monero-core_it.ts [hunk 15]
```diff
@@ -847,83 +848,83 @@
...
 Payment ID: </source>
-        <translation type="unfinished"></translation>
+        <translation>ID Pagamento: </translation>
     </message>
     <message>
         <location filename="../main.qml" line="364"/>
         <source>
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>Totlae: </translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: Totale:

File: translations/monero-core_it.ts (hunk 1 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 3 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 4 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 7 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 8 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 9 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 10 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Si giusto, correggo

File: translations/monero-core_it.ts (hunk 12 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 14 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: translations/monero-core_it.ts (hunk 15 above)
Comment (signo88) [inline translations/monero-core_it.ts]: Correggo

File: lang/languages.xml [hunk 16]
```diff
@@ -16,9 +16,9 @@ List of available languages for your wallet's seed:
         <language display_name="Russia"     locale="ru_RU" wallet_language="Russian" flag="/lang/flags/russia.png" qs="none"/>
         <language display_name="RPA"        locale="en_SA" wallet_language="English" flag="/lang/flags/rpa.png" qs="none"/>
         <language display_name="Palestine"  locale="ar_PS" wallet_language="English" flag="/lang/flags/palestine.png" qs="none"/>
-        <language display_name="India"      locale="hi_IN" wallet_language="English" flag="/lang/flags/india.png" qs="none"/>
+        <language display_name="India"      locale="hi_IN" wallet_language="English" flag="/lang/flags/india.png" qs="none"/> -->
         <language display_name="Italy"      locale="it_IT" wallet_language="Italian" flag="/lang/flags/italy.png" qs="none"/>
```
Comment (Jaqueeee) [inline lang/languages.xml]: Translation is not fully implemented yet, so please remove this.

Comment (signo88) [inline lang/languages.xml]: Done..

**Comment by dEBRUYNE-1** [comment]
Perhaps we should rename "Italy" to "Italian" in the wizard? Any opinions @iDunk5400 and @signo88?

**Comment by maitscha** [comment]
IMHO on a language selection page the languages should be in their native language. So "Italian" would be "Italiano". If the user can't speak englisch, he would still find his language. "Deutsch" instead of "German" and so on ...

**Comment by signo88** [comment]
I agree with @maitscha. We should consider to rename "Italy" with "Italiano".

File: translations/monero-core_it.ts [hunk 17]
```diff
@@ -159,89 +159,89 @@
...
         <location filename="../pages/History.qml" line="65"/>
         <source>Filter transaction history</source>
-        <translation type="unfinished"></translation>
+        <translation>Filtra lo storico transazioni</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="102"/>
         <source>&lt;b&gt;Payment ID&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;A unique user name used in&lt;br/&gt;the address book. It is not a&lt;br/&gt;transfer of information sent&lt;br/&gt;during thevtransfer</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;ID Pagameto&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;Utente unico usato nella&lt;br/&gt;rubrica.</translation>
```
Comment (iDunk5400) [inline translations/monero-core_it.ts]: ID Pagamento  :)

**Comment by iDunk5400** [comment]
Makes sense.

**Comment by dEBRUYNE-1** [comment]
@maitscha, coincidentally I was thinking about that after I made the communt, thus ACK for that concept.

**Comment by Jaqueeee** [comment]
@signo88  More strings have been added to GUI since you added this. Please update translation when you have time.

**Pull request merged**